### PR TITLE
ramips: use led, function/color syntax for some routers

### PR DESCRIPTION
**Pull request by namiltd**
- Rename the mt7621_dlink_dir-xx60-a1.dtsi file to a more appropriate name.
- Abandoning the label and using the function/color syntax for some dlink dir series routers: dir-1960-a1, dir-2660-a1, dir-2640-a1, dir-3040-a1 and dir-3060-a1
- Setting the LED name and abandoning the label and using the function/color syntax for some TP-Link Archer series routers: Archer C2 v1, Archer C20 v1, Archer C20i and Archer C50 v1

### CODE REVIEW

**Comment by Lucky1openwrt** [comment]
ok cool I guess with v24 if you sysupgrade you have to fix the leds with the version change
I think you have to change the aliases as well tho
```

	aliases {
		label-mac-device = &gmac0;
		led-boot = &led_power_orange;
		led-failsafe = &led_power_white;
		led-running = &led_power_white;
		led-upgrade = &led_net_orange;
	};
```

**Comment by namiltd** [comment]
Migration done.

**Comment by Lucky1openwrt** [comment]
all good :)
gave it a test run on my DIR-1960-A1 
all worked even the Migration :)

File: target/linux/ramips/image/mt7621.mk [hunk 1]
```diff
@@ -763,35 +763,35 @@ endef
...
-  $(Device/dlink_dir-xx60-a1)
+  $(Device/dlink_dir_nand_128m)
   DEVICE_MODEL := DIR-3040
   DEVICE_VARIANT := A1
 endef
 TARGET_DEVICES += dlink_dir-3040-a1
 
 define Device/dlink_dir-3060-a1
-  $(Device/dlink_dir-xx60-a1)
+  $(Device/dlink_dr_nand_128m)
```
Comment (Lucky1openwrt) [inline target/linux/ramips/image/mt7621.mk:794]: missing an i in $(Device/dlink_dr_nand_128m)

Comment (namiltd) [reply target/linux/ramips/image/mt7621.mk]: Fixed.

**Review by Lucky1openwrt — APPROVED** [review]

**Comment by namiltd** [comment]
Add in description: Successor of PR https://github.com/openwrt/openwrt/pull/15275

**Pull request merged**